catalog/resource_grants.go

```diff
@@ -102,6 +102,9 @@ func getPermissionEndpoint(securable, name string) string {
 	if securable == "share" {
 		return fmt.Sprintf("/unity-catalog/shares/%s/permissions", name)
 	}
+	if securable == "the_connection" {
```

Comment:
is `the_connection` correct name for securable?

Reply:
typo 😶‍🌫️

Reply:
I've added a test to confirm